src/kibana/index.html

```diff
@@ -7,6 +7,22 @@
     <meta name="viewport" content="width=device-width">
     <link rel="shortcut icon" href="styles/theme/elk.ico">
     <title>Kibana 4</title>
+    <link rel="stylesheet" href="styles/main.css?_b=@@buildNum">
+
+  </head>
+  <body kibana ng-class="'application-' + activeApp.id">
+
+    <div class="row">
+      <div class="col-md-offset-4 col-md-4 page-header initial-load">
+        <center>
+          <img width="128" src="images/initial_load.gif">
```

Comment:
You didn't check in the loading images. Otherwise, things look fine.

I plugged in ![gif](http://media.giphy.com/media/4c4jQPdVv381G/giphy.gif) for testing ;)